tools/gatk_getpileupsummaries.cwl

```diff
@@ -8,14 +8,14 @@ requirements:
   - class: DockerRequirement
     dockerPull: 'kfdrc/gatk:4.1.1.0'
   - class: ResourceRequirement
-    ramMin: 2000
+    ramMin: ${ return inputs.max_memory * 1000 }
```

Comment:
If this is a ram min, why is it referencing the max_memory parameter?

Reply:
Because the value is used to set the max memory in the java options.